department-of-veterans-affairs/caseflow
# Artem/new v2 faq

connects #2087

- Shows updated Section 5 and 6.1 for Certification v2
- Old Section 5 and 6.1 will still show up without `FeatureToggle` for `:certification_v2`

### Steps to validate:

1. Enable `:certification_v2` FeatureToggle
1. Visit `certification/help` and check out section 5 and 6.1 see *new copy*
1. Disable `:certification_v2` FeatureToggle
1. Visit `certification/help` and check out section5 and 6.1 see *old copy*
1. Visit `certifications/new/456C` and click **Learn more about document dates.**

The diff looks all messed up because I also fixed the indentation for clarify of parsing.

## OLD:
<img width="930" alt="screen shot 2017-06-09 at 2 57 03 pm" src="https://user-images.githubusercontent.com/18075411/26990267-1293d98a-4d24-11e7-8b69-c9b02d96629c.png">

## NEW:
<img width="925" alt="screen shot 2017-06-09 at 2 57 57 pm" src="https://user-images.githubusercontent.com/18075411/26990286-1dfbcb34-4d24-11e7-951b-2f2020189ec9.png">
